src/styles/_catalog.scss

```diff
@@ -1,22 +1,27 @@
 .catalog {
```

Comment:
this file should have name `catalog.scss`, `__` prefix is more about utility files, they are not BEM-related